index.html

```diff
@@ -17,7 +18,10 @@
   <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"/>
   <link rel="stylesheet" href="./CSS/style.css" />
 </head>
+
 <body>
+  <!-- Start of Header -->
```

Comment:
- [ ] Please remove all commented codes, to have a clean code.

Reply:
Thanks